.rubocop.yml

```diff
@@ -47,3 +47,14 @@ Style/SymbolArray:
 
 Style/WordArray:
   EnforcedStyle: brackets
+
+# In rspec, we use blocks everywhere -- this rule is always fired off
+# because  the initial describe wraps so much. Let's just disable it for
+# specs.
+Metrics/BlockLength:
+  Exclude:
+    - 'spec/**/*.rb'
+
+RSpec/ExampleLength:
```

Comment:
By default, the max number of lines that `rubocop-rspec` likes is 5 per spec (`it` block). This change is to increase to 25 lines, and to count multi-line array/hash/heredoc strings as a single line.